astro/src/content/quickstarts/quickstart-php-web.mdx

```diff
@@ -0,0 +1,300 @@
+
+You will run PHP 8, Composer, and Apache in Docker. Using Docker makes this tutorial suitable for Windows, Linux, and macOS.
+
+If you are experienced in PHP, you can install PHP 8, Composer, and Apache on your host machine instead and convert commands to work on your operating system.
+
+## General Architecture
+
+This sample application doesn't have login functionality without FusionAuth, but a more typical integration will replace an existing login system with FusionAuth.
+
+In that case, the system might look like this before FusionAuth is introduced.
+
+<LoginBefore alt={"Request flow during login before FusionAuth"}/>
+
+The login flow will look like this after FusionAuth is introduced.
+
+<LoginAfter alt={"Request flow during login after FusionAuth"}/>
+
+In general, you would introduce FusionAuth to normalize and consolidate user data to make it consistent and up to date, and offload your login security and functionality to FusionAuth.
+
+## Getting Started
+
+Start with getting FusionAuth up and running and creating a new {frontmatter.technology} application.
+
+### Clone The Code
+
+First, grab the code from the repository and change to that directory.
+
+```shell
+git clone https://github.com/FusionAuth/fusionauth-quickstart-php-web.git
+cd fusionauth-quickstart-php-web
+mkdir your-application
+```
+
+All shell commands in this guide can be entered in a terminal in this directory. On Windows, you need to replace forward slashes with backslashes in paths.
+
+All the files you'll create in this guide already exist in the `complete-application` subdirectory, if you prefer to copy them to `your-application`.
+
+### Run FusionAuth Via Docker
+
+You'll find a Docker Compose file `docker-compose.yml` and an environment variables configuration file `.env` in the root directory of the repository.
+
+Assuming you have Docker installed, you can start FusionAuth on your machine with the following.
+
+```shell
+docker compose up
+```
+
+This will start four containers, one each for FusionAuth, Postgres, OpenSearch, and PHP.
+
+Here you are using a bootstrapping feature of FusionAuth called [Kickstart](/docs/v1/tech/installation-guide/kickstart). When FusionAuth starts for the first time, it will look at the `kickstart/kickstart.json` file and configure FusionAuth to your specified state.
+
+<Aside type="note">
+  If you ever want to reset the FusionAuth system, delete the volumes created by `docker-compose` by executing `docker
+  compose down -v`, then rerun `docker compose up -d`.
+</Aside>
+
+FusionAuth will be configured with these settings:
+
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
```

Comment:
```suggestion
* Your client Id is <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.variables.applicationId" /></code>.
* Your client secret is <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.requests.[?(@.url === '/api/application/#{applicationId}')].body.application.oauthConfiguration.clientSecret" /></code>.
* Your example username is <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.variables.userEmail" /></code> and the password is <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.variables.userPassword" /></code>.
* Your admin username is <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.variables.adminEmail" /></code> and the password is <code><RemoteValue url={frontmatter.codeRoot + "/kickstart/kickstart.json"} selector="$.variables.adminPassword" /></code>.
```

Using [`<RemoteValue>`](https://github.com/FusionAuth/fusionauth-astro-components#remotevalue) to avoid hard-coding these values